.npmrc

```diff
@@ -1,8 +1,8 @@
 # username: prclt-frontend 2019-02-01T00:05:50.843Z
 # allows access to restricted @percolate/* packages
-//registry.npmjs.org/:_authToken=e5cf8d69-5a56-4cbf-8e3f-d268e6d21b69
+//registry.npmjs.org
```

Comment:
just remove this completely. Have it so just `registry=` is defined and that is it.

Basically we are adding this here to make sure that this repo defaults to the yarn registry. If someone has a local npmrc, like we do, that points to jfrog as the registry, it will update the yarn lock to use that registry which is not what we want. We want this repo to always use the one defined here.